talks/fr.json

```diff
@@ -0,0 +1,16 @@
+{
+	"fromWhatCurrency": [
+		"Quelle devise souhaitez-vous convertir ?",
+		"Quelle est la devise ?",
+		"Bien, mais quelle est la devise ?"
+	],
+	"noServer": [
+		"Malheureusement, je ne peux pas atteindre les dernières valeurs, veuillez réessayer dans un moment."
+	],
+	"noApiKey": [
+		"Je peux faire cela sans une clef de l'API. Veuillez l'ajouter à ma configuration."
+	],
+	"answer": [
+		"{} {} is {} {}"
```

Comment:
I think you forgot to translate this sentence. When alice answers it will replace the fields with the values and currencies. @Psychokiller1888 are changes in the code required to make it work with different languages than en?

Reply:
@LeTengu Merci pour la traduction. Par contre => je NE peux pas faire cela sans une clef